config/icd11.sssom-metadata.yml

```diff
@@ -0,0 +1,15 @@
+creator_id: orcid:0000-0002-2906-7319
+curie_map:
+  icd11.foundation: http://id.who.int/icd/entity/
+  MONDO: http://purl.obolibrary.org/obo/MONDO_
+  oboInOwl: http://www.geneontology.org/formats/oboInOwl#
+  orcid: https://orcid.org/
+  owl: http://www.w3.org/2002/07/owl#
+  Orphanet: http://www.orpha.net/ORDO/Orphanet_
+  rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+  rdfs: http://www.w3.org/2000/01/rdf-schema#
+  semapv: https://w3id.org/semapv/
+  skos: http://www.w3.org/2004/02/skos/core#
+  sssom: https://w3id.org/sssom/
+license: http://w3id.org/sssom/license/unspecified
+mapping_provider: https://www.orpha.net/
```

Comment:
## `mapping_provider` for **Mondo -(ORDO)--> ICD11**
- [ ] @matentzn advise

We're doing `mapping_provider: https://www.orpha.net/` for the ORDO->ICD11 mappings, but the MONDO->ICD11 mappings require inputs from Orphanet as well as `mondo.sssom.tsv`. I've left the `mapping_provider` the same, but I have a feeling it should be changed to something else.

Reply:
Indeed probably a composite `mapping_provider` would make sense, but not to worry.